docs/tsc/meetings/2019-02-25.md

```diff
@@ -0,0 +1,57 @@
+Feb 25, 2019
+
+Host: Michael Dolan
+
+Rotating Secretary: Mark Boorer
+
+Attendees:
+  * [X] Mark Boorer (_TSC_) - Industrial Light & Magic
+  * [X] Sean Cooper (_TSC_) - DNEG
+  * [X] Michael Dolan (_TSC Chair_) - Sony Pictures Imageworks
+  * [X] Larry Gritz (_TSC_) - Sony Pictures Imageworks
+  * [X] Thanh Ha - Academy Software Foundation / Linux Foundation
+  * [X] Patrick Hodoul (_TSC_) - Autodesk
+  * [ ] John Mertic - Academy Software Foundation / Linux Foundation
+  * [ ] Carl Rand (_TSC_) - Foundry
+  * [X] Doug Walker (_TSC_) - Autodesk
+  * [X] Kevin Wheatley (_TSC_) - Framestore
+
+# **OCIO TSC Meeting Notes**
+
+* CI
+    - Thinking is that because ASWF already has Linux CI support, we could start using it.
+    - The ASWF solution is Jenkins at the moment, but due to the young age of the project, no-one is really using it yet.
+    - Circle CI has been proposed as a potential candidate as well.
+    - There is a proof of concept of OCIO on the travis CI, though it is only a simple CMake build (no tests).
+    - Additionally it would be good to get away from Travis CI as its future seems uncertain.
+    - Potentially wait for results of ASWF TAC discussion on future CI decision before actioning any changes.
+    - Thoughts are that migration of repo to ASWF is more important. We can keep our existing CI, and just move the repo.
+        - **TODO**: Might need someone from Sony to do the ASWF move.
+        - There will be a day or so crossover to set up the existing CI on the new ASWF setup.
+        - Decided to move to ASWF first, then sort CI later.
+    - **TODO**: Candidate for championing CI? Mark Boorer and Sean Cooper.
+
+* Embedded libraries
+    - Decision to remove all embedded dependencies at the same time as CI update.
+    - Concerns over Windows developer experience (as building of dependencies would now be required prior to building OCIO.
+    - Potentially solvable via vcpkg or Chocolatey.
+
+* 1.1 patch release
+    - Updates to OCIO and OIIO to allow c++11 / c++98 link time compatibility.
```

Comment:
[Comment] Even if all compilers claim to be compatible I stay a way from that kind of binary mixes. My 2 cents.

Reply:
Perhaps it would be better worded:

Updates to OCIO to allow optional OIIO 2.0 compatibility (which requires C++11).